pysd/py_backend/utils.py

```diff
@@ -262,14 +262,14 @@ def make_python_identifier(string, namespace=None, reserved_words=None,
 
     if convert == 'hex':
         # Convert invalid characters to hex
-        s = ''.join([c.encode("hex") if re.findall('[^0-9a-zA-Z_]', c) else c for c in s])
+        s = ''.join([c.encode("hex") if re.findall('[^\p{l}\p{m}\p{n}_]', c) else c for c in s])
```

Comment:
I'm not sure that this regex is a correct... By regex101 service it match any sequence...

Reply:
I think it is fine; it uses unicode character blocks (which regex101.com does not understand, nor does the python re module, which is why I had to switch to the regex module). It should not match = ( ) and other non-python identifier sequences.Which regex service are you using?

Reply:
Thanks for the revisions!

Reply:
Yes, I'm use regex in python mode to verify that.. But I'm completelly understand this syntax of regex... It will be good to add some comments and add additional tests to check that nothing is broken.

Reply:
I added comments to describe what this regex is doing. As for additional tests, I think that the Unicode (and non-unicode) tests already present should verify that this works?